ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -0,0 +1,213 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Need a test for this file. It's complicated with a number of parameters, and specifically you need to really ensure the manifest file system works properly